src/Proj4.jl

```diff
@@ -1,5 +1,8 @@
 module Proj4
 
+using BinDeps
+@BinDeps.load_dependencies
+
 const libproj = "libproj"
```

Comment:
Should we now remove the `const libproj = "libproj"` from here, as `BinDeps` will define it for us in `load_dependencies`?